README.md

```diff
@@ -182,18 +186,18 @@ distro.Run()
 otel.SetTextMapPropagator(propagation.TraceContext{})
 ```
 
-## Correlate Traces and Logs
+## Correlate traces and logs
 
 You can add trace metadata to logs using the OpenTelemetry trace API. Trace
 metadata lets you explore logs in Splunk Observability Cloud.
 
-See [Correlating Trace and Logs](./docs/correlating-trace-and-logs.md) for more
+See [Correlating Trace and Logs](https://docs.splunk.com/Observability/gdi/get-data-in/application/go/instrumentation/connect-traces-logs.html) for more
```

Comment:
I do not like this hyperlinks change 😋 

Maybe we should change `README.md` to just have e.g.:

```
# Splunk Distribution of OpenTelemetry Go

[![Splunk GDI Specification](https://img.shields.io/badge/GDI-1.2.0-blue)](https://github.com/signalfx/gdi-specification/releases/tag/v1.2.0)
[![GitHub Release](https://img.shields.io/github/v/release/signalfx/splunk-otel-go?include_prereleases)](https://github.com/signalfx/splunk-otel-go/releases)
[![Go Reference](https://pkg.go.dev/badge/github.com/signalfx/splunk-otel-go.svg)](https://pkg.go.dev/github.com/signalfx/splunk-otel-go)
[![go.mod](https://img.shields.io/github/go-mod/go-version/signalfx/splunk-otel-go)](go.mod)
[![LICENSE](https://img.shields.io/github/license/signalfx/splunk-otel-go)](LICENSE)
[![Build Status](https://img.shields.io/github/workflow/status/signalfx/splunk-otel-go/ci)](https://github.com/signalfx/splunk-otel-go/actions?query=branch%3Amain)
[![codecov](https://codecov.io/gh/signalfx/splunk-otel-go/branch/main/graph/badge.svg)](https://codecov.io/gh/signalfx/splunk-otel-go)
[![Go Report Card](https://goreportcard.com/badge/github.com/signalfx/splunk-otel-go)](https://goreportcard.com/report/github.com/signalfx/splunk-otel-go)

The Splunk distribution of [OpenTelemetry
Go](https://github.com/open-telemetry/opentelemetry-go) provides
multiple packages that automatically instruments your Go
application to capture and report distributed traces to Splunk APM.

The official documentation for this distribution can be found here:

https://docs.splunk.com/Observability/gdi/get-data-in/application/go/get-started.html

> :construction: This project is currently in **BETA**.
> It is **officially supported** by Splunk.
> However, breaking changes **MAY** be introduced.

If you're currently using the [SignalFx Tracing Library for Go](https://github.com/signalfx/signalfx-go-tracing)
and want to migrate to the
Splunk Distribution of OpenTelemetry Go, see [Migrate from the SignalFx Go
Agent](https://docs.splunk.com/Observability/gdi/get-data-in/application/go/troubleshooting/migrate-signalfx-go-to-otel.html).
```

Then we could move the existing content (and keep the relative hyperlinks) to `docs/README.md`.

WDYT?

Reply:
Agree!